.github/workflows/nightly-windows-ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           - 'v142'
```

Comment:
Will we change v141 to v143?

Reply:
Yes, I missed this one. Addressed in 11406f9